New dfttk.vasp_input.py

nhew1994 opened the pull request:
To generate VASP input files for workflows.py

nhew1994 committed:
New dfttk.vasp_input.py
To generate VASP input files for workflows.py

nhew1994 committed:
Update dfttk/vasp_input.py
Removed ISIF parameter from ev_curve_set. Using other_settings is enough.

lukeamyers reviewed (approved):
nice. just minor suggestions

nhew1994 committed:
Update dfttk.vasp_input.py
potcar_file -> potcar_functional
functional -> incar_functional
poscar -> struct
semicond_or_insul -> non_metal
